.devcontainer/devcontainer.json

```diff
@@ -22,7 +22,7 @@
 {
 	"name": "GraphAr",
 	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
-	"image": "ghcr.io/apache/graphar-dev:latest",
```

Comment:
the image name is [graphar-dev](https://github.com/apache/incubator-graphar/pkgs/container/graphar-dev)